packages/fxa-auth-server/lib/subhub.js

```diff
@@ -0,0 +1,315 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const isA = require('joi');
+const error = require('./error');
+const createBackendServiceAPI = require('./backendService');
+
+/*
+ * The subscriptions backend is called 'SubHub', a service managed outside the
+ * FxA team to abstract away some details of payments processing.
+ *
+ * This library implements a little proxy in front of the SubHub API, allowing
+ * it to be authenticated by FxA's bearer token.
+ */
+
+module.exports = function (log, config) {
+  if (config.subhub.useStubs) {
+    // TODO: Remove this someday after subhub is available
+    return buildStubAPI(log, config);
+  }
+
+  if (! config.subhub.enabled) {
+    return [
+      'listPlans',
+      'listSubscriptions',
+      'createSubscription',
+      'getCustomer',
+      'updateCustomer',
+      'cancelSubscription',
+    ].reduce((obj, name) => ({
+      ...obj,
+      [ name ]: () => Promise.reject(error.featureNotEnabled())
+    }), {});
+  }
+
+  const SubHubAPI = createBackendServiceAPI(log, config, 'subhub', {
```

Comment:
As an aside, I'm happy to see this being useful for additional backend service calls :-)